### PR TITLE
Octarine bugfix

### PR DESCRIPTION
**Pull request by nlsun**
Octarine tests were hitting the wrong url, this was causing tests to pass even though it was misbehaving. Also updated Octarine to fix this.

### CODE REVIEW

**Comment by nlsun** [comment]
Ready for review. I think my privileges got reset so I can't restart that 1 failed build.

File: packages/dcos-integration-test/extra/test_applications.py [hunk 1]
```diff
@@ -68,7 +68,8 @@ def test_octarine_http(cluster, timeout=30):
     octarine_id = uuid.uuid4().hex
     proxy = ('"http://127.0.0.1:$(/opt/mesosphere/bin/octarine ' +
              '--client --port {})"'.format(octarine_id))
-    check_command = 'curl --fail --proxy {} marathon.mesos'.format(proxy)
+    url = "marathon.mesos.mydcos.directory"
```
Comment (cmaloney) [inline packages/dcos-integration-test/extra/test_applications.py]: given this is a locally used only thing, why not just embed it as a constant in the check_command string?

File: packages/dcos-integration-test/extra/test_applications.py [hunk 2]
```diff
@@ -107,7 +108,8 @@ def test_octarine_srv(cluster, timeout=30):
     cmd = ('/opt/mesosphere/bin/octarine {} & '.format(octarine_id) +
            '/opt/mesosphere/bin/python -m http.server ${PORT0}')
     raw_app_id = 'integration-test-app-octarine-srv-{}'.format(test_uuid)
-    check_command = ('curl --fail --proxy {} _{}._{}._tcp.marathon.mesos')
+    check_command = ('curl --fail --proxy {} ' +
```
Comment (cmaloney) [inline packages/dcos-integration-test/extra/test_applications.py]: Our wrap width is 120 characters, this is under that / doesn't need to wrap

To match the rest of the codebase should probably be:
```python
    check_command = 'curl --fail --proxy {} _{}._{}._tcp.marathon.mesos.mydcos.directory'.format(
        proxy
        port_name
        raw_app_id)
```

**Comment by cmaloney** [comment]
How did you find this was off / is there some more end to end integration test (like using `dcos-tunnel` or the like) which we could use for this longer term?

**Comment by cmaloney** [comment]
:ship:

**Comment by nlsun** [comment]
I was actually doing a full demo for documentation purposes and that's when I noticed that it wasn't working.

I have a test that would have caught this that runs when I push to the CLI module part of this, so perhaps that could be added to CI.

**Pull request merged**